README.md

```diff
@@ -80,7 +80,7 @@ versions of InstrumentStudio.
 
 | InstrumentStudio Version | Release | Download |
 | ------------------------ | -------- | ------- |
-| 2024 Q3                  | [2.0.0](https://github.com/ni/measurement-plugin-python/releases/tag/2.0.0) | [measurement-plugin-python-examples-2.0.0.zip](https://github.com/ni/measurement-plugin-python/releases/download/2.0.0/measurement-plugin-python-examples-2.0.0.zip) |
+| 2024 Q3                  | [2.1.0](https://github.com/ni/measurement-plugin-python/releases/tag/2.1.0) | [measurement-plugin-python-examples-2.1.0.zip](https://github.com/ni/measurement-plugin-python/releases/download/2.1.0/measurement-plugin-python-examples-2.1.0.zip) |
```

Comment:
We should be adding 2024 Q4 with the 2.1.0 version, not replacing 2024 Q3.

Reply:
Updated. Should I also remove v1.4 as it is old and anyway present in "Older versions" link?

Reply:
I think we can leave it for now.